slave-services/src/main/java/org/iqu/slaveservices/entities/Events.java

```diff
@@ -3,31 +3,35 @@
 import java.util.HashSet;
 import java.util.Set;
 
```

Comment:
Probably this class isn't needed anymore since we have org.iqu.slaveservices.rest.consumer.Events